tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import pytest
+import time
+
+from common.fixtures.conn_graph_facts import fanout_graph_facts
+from common.helpers.assertions import pytest_assert
+from common.helpers.pfc_storm import PFCStorm
+from files.pfcwd_helper import start_wd_on_ports
+
+pytestmark = [pytest.mark.disable_loganalyzer]
```

Comment:
why disable loganalyzer here?

Reply:
did it out of habit because I was explicitly using the loganalyzer in the other 2 tests. will remove it